Table: `pReorderableRowHandle` directive generates `error TS2322: Type 'string' is not assignable to type 'number'`

### Describe the bug

With `tsconfig.json` > `angularCompilerOptions.strictTemplates: true`, including `pReorderableRowHandle` on a production build fails with `error TS2322: Type 'string' is not assignable to type 'number'`

### Environment

See below.

### Reproducer

_No response_

### Angular version

14.2.7

### PrimeNG version

14.1.2

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

16.17.0

### Browser(s)

N/A

### Steps to reproduce the behavior

1. Use `pReorderableRowHandle` in a template containing Table as shown in [the PrimeNG showcase](https://www.primefaces.org/primeng/table/reorder) in an Angular project with `tsconfig.json` > `angularCompilerOptions.strictTemplates: true`
2. Perform a production build

### Expected behavior

The application should compile without error.

--- a/src/app/components/table/table.ts
+++ b/src/app/components/table/table.ts
@@ -4086,8 +4086,6 @@ export class TableHeaderCheckbox {
     }
 })
 export class ReorderableRowHandle implements AfterViewInit {
-    @Input('pReorderableRowHandle') index: number;
-
     constructor(public el: ElementRef) {}
 
     ngAfterViewInit() {

